coalib/bears/Bear.py

```diff
@@ -24,6 +24,26 @@
 from .meta import bearclass
 
 
+def _debugger_setting_value(bear):
+    """
+    Determines the value of ``debugger`` setting.
+
+    :param bear: Bear object.
+    :return:     True if ``debug_bears`` is ``True`` or if bear name specified
+                 in ``debug_bears`` setting match with the bear parameter.
+    """
+    if bear is None:
+        return False
+    if 'debug_bears' not in bear.section:
+        return False
+    try:
+        return bool(bear.section['debug_bears'])
+    except ValueError:
+        pass
+    return bear.name.lower() in (
+            map(str.lower, list(bear.section['debug_bears'])))
```

Comment:
Is using `list` necessary? `map` tries to create an iterator as well out of `bear.section[...]` if I'm not mistaken